services/velero/metadata.yaml

```diff
@@ -2,6 +2,8 @@ displayName: Velero
 description: Open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.
 category:
   - backup
+dependencies:
+  - rook-ceph-cluster
```

Comment:
same as Loki, it's possible to deploy Velero without rook-ceph but it requires the user to configure Velero correctly. 

does `dependencies` mean we just show a warning?

Reply:
Yep, the user will see a modal asking them to proceed.